starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -62,6 +62,7 @@
 _DEFAULT_PLATFORM = 'evergreen-x64'
 _DEFAULT_CONFIG = 'gold'
 _DEFAULT_TARGET = 'cobalt'
+_DEFAULT_SB_VERSION = '16'
```

Comment:
Change this to an int and don't cast it when you use it, instead use the `type` parameter in the call to add_argument for the parser